Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -868,27 +875,53 @@ def fetch_incidents(client, last_run, first_fetch_time, min_severity):
     # Get the last fetch details, if exist
     last_fetch_time = last_run.get('last_fetch_time')
     last_fetch_ids = last_run.get('last_fetch_ids', [])
+    last_incident_number = last_run.get('last_incident_number')
+    demisto.debug(f"last fetch time is: {last_fetch_time}, last fetch ids are:{last_fetch_ids}")
```

Comment:
```suggestion
    demisto.debug(f"{last_fetch_time=}, {last_fetch_ids=}")
```
Sweet sweet python 3.8 syntax sugaring